VAMobile/src/constants/linking.tsx

```diff
@@ -15,6 +21,8 @@ export const linking: LinkingOptions<any> = {
         screens: {
           HealthTab: {
             screens: {
+              ClaimsHistory: 'claims',
+              PrescriptionHistory: 'prescriptions',
```

Comment:
This can be left out of the PR. I'm planning on removing this entirely from the feature-xl branch, since these links are handled in `getStateFromPath`